src/locales/cy/fields.yml

```diff
@@ -0,0 +1,6 @@
+nationalInsuranceNumber:
+  content: " "
+  label: What is your national insurance number?
```

Comment:
This is in English at the moment - translations are expected next week, and this isn't live yet, so it's safe to merge with the wrong language in it - for now.